addons/account/models/reconciliation_widget.py

```diff
@@ -529,8 +574,8 @@ def _prepare_move_lines(self, move_lines, target_currency=False, target_date=Fal
 
             debit = line.debit
             credit = line.credit
-            amount = line.amount_residual
-            amount_currency = line.amount_residual_currency
+            amount = line.balance if to_check else line.amount_residual
+            amount_currency = line.amount_currency if to_check else line.amount_residual_currency
```

Comment:
o_O

not sure at all, but this needs testing for confirmation

Reply:
Maybe this isn't how I should do it, but amount_residual is 0 because the line is already reconciled. I will call button_cancel_reconciliation right after, but before it is done, I need to know the balance of the line